src/services/ui/src/pages/actions/WithdrawPackage.tsx

```diff
@@ -84,162 +85,83 @@ const WithdrawPackageForm: React.FC = ({ item }: { item?: ItemResult }) => {
+            not be able to resubmit this package. CMS will be notified and will
+            use this content to review your request. If CMS needs any additional
+            information, they will follow up by email.
+          </p>
+        </ActionFormIntro>
+        <PackageInfo item={item} />
+        <I.Form {...form}>
+          <form onSubmit={form.handleSubmit(handleSubmit)}>
+            {/* Change faqLink once we know the anchor */}
+            <AttachmentsSizeTypesDesc faqLink={"/faq"} />
+            {attachments.map(({ name, label, required }) => (
               <I.FormField
+                key={name}
                 control={form.control}
-                name="additionalInformation"
+                name={`attachments.${name}`}
                 render={({ field }) => (
                   <I.FormItem className="mt-8">
-                    <h3 className="font-bold text-2xl font-sans">
-                      Additional Information
-                    </h3>
-                    <I.FormLabel className="font-normal">
-                      Explain your need for withdrawal or upload supporting
-                      documentation.
-                      <br />
-                      <p>
-                        <em className="italic">
-                          Once you submit this form, a confirmation email is
-                          sent to you and to CMS. CMS will use this content to
-                          review your package. If CMS needs any additional
-                          information, they will follow up by email.
-                        </em>{" "}
-                      </p>
-                      <br />
+                    <I.FormLabel>
+                      {label}
+                      {required ? <I.RequiredIndicator /> : ""}
                     </I.FormLabel>
-                    <I.Textarea {...field} className="h-[200px] resize-none" />
-                    <I.FormDescription>
-                      4,000 characters allowed
-                    </I.FormDescription>
+                    <I.Upload
+                      files={field?.value ?? []}
+                      setFiles={field.onChange}
+                    />
+                    <I.FormMessage />
                   </I.FormItem>
                 )}
               />
-              {errorMessage && (
-                <div className="text-red-500 mt-4">{errorMessage}</div>
+            ))}
+            <I.FormField
+              control={form.control}
+              name="additionalInformation"
+              render={({ field }) => (
+                <I.FormItem className="mt-8">
+                  <h3 className="font-bold text-2xl font-sans">
+                    Additional Information
+                  </h3>
+                  <I.FormLabel className="font-normal">
+                    Explain your need for withdrawal or upload supporting
+                    documentation.
+                    <br />
```

Comment:
This is just a personal thing, but is there anyway we can forego the use of a `<br />` tag for something else, whether that be through the use of a flex-box, margin or padding?

Reply:
I am also recognizing that it may have been there already, and if that is the case you can ignore the previous comment

Reply:
See above :)

Reply:
Gotcha.